po/fr.po

```diff
@@ -1,5 +1,723 @@
+msgid "Open Akira file"
+msgstr "Ouvrir un fichier Akira"
+
+#: src/FileFormat/FileManager.vala:105 src/Layouts/HeaderBar.vala:170
+msgid "Open"
+msgstr "Ouvrir"
+
+#: src/Layouts/HeaderBar.vala:69 src/Lib/Managers/ExportManager.vala:296
+msgid "Untitled"
+msgstr "Sans titre"
+
+#: src/Layouts/HeaderBar.vala:71
+msgid "Menu"
+msgstr "Menu"
+
+#: src/Layouts/HeaderBar.vala:75
+msgid "Insert"
+msgstr "Insérer"
+
+#: src/Layouts/HeaderBar.vala:82
+msgid "Group"
+msgstr "Grouper"
+
+#: src/Layouts/HeaderBar.vala:84
+msgid "Ungroup"
+msgstr "Dégrouper"
+
+#: src/Layouts/HeaderBar.vala:87
+msgid "Up"
+msgstr "Monter"
+
+#: src/Layouts/HeaderBar.vala:92
+msgid "Down"
+msgstr "Descendre"
+
+#: src/Layouts/HeaderBar.vala:97
+msgid "Top"
+msgstr "Devant"
+
+#: src/Layouts/HeaderBar.vala:102
+msgid "Bottom"
+msgstr "Derrière"
+
+#: src/Layouts/HeaderBar.vala:107
+msgid "Settings"
+msgstr "Paramètres"
+
+#: src/Layouts/HeaderBar.vala:118
+msgid "Difference"
+msgstr "Différence"
+
+#: src/Layouts/HeaderBar.vala:120
+msgid "Exclusion"
+msgstr "Exclusion"
+
+#: src/Layouts/HeaderBar.vala:122
+msgid "Intersect"
+msgstr "Intersection"
+
+#: src/Layouts/HeaderBar.vala:124
+msgid "Union"
+msgstr "Union"
+
+#: src/Layouts/HeaderBar.vala:161
+msgid "New Window"
+msgstr "Nouvelle fenêtre"
+
+#: src/Layouts/HeaderBar.vala:182
+msgid "Open Recent"
+msgstr "Ouvert récemment"
+
+#: src/Layouts/HeaderBar.vala:195
+msgid "Save As"
+msgstr "Enregistrer sous..."
+
+#: src/Layouts/HeaderBar.vala:204
+msgid "Quit"
+msgstr "Quitter"
+
+#: src/Layouts/HeaderBar.vala:237
+msgid "Artboard"
+msgstr "Plan de travail"
+
+#: src/Layouts/HeaderBar.vala:254
+msgid "Add Items"
+msgstr "Ajouter des éléments"
+
+#: src/Layouts/HeaderBar.vala:262
+msgid "Rectangle"
+msgstr "Rectangle"
+
+#: src/Layouts/HeaderBar.vala:268
+msgid "Ellipse"
+msgstr "Ellipse"
+
+#: src/Layouts/HeaderBar.vala:286
+msgid "Vector"
+msgstr "Vecteur"
+
+#: src/Layouts/HeaderBar.vala:288
+msgid "Pencil"
+msgstr "Crayon"
+
+#: src/Layouts/HeaderBar.vala:291
+msgid "Text"
+msgstr "Texte"
+
+#: src/Layouts/HeaderBar.vala:297
+msgid "Image"
+msgstr "Image"
+
+#: src/Layouts/HeaderBar.vala:330
+msgid "Export Current Selection"
+msgstr "Exporter la sélection courante"
+
+#: src/Layouts/HeaderBar.vala:336
+msgid "Export Artboards"
+msgstr "Exporter les plans de travail"
+
+#: src/Layouts/HeaderBar.vala:345
+msgid "Highlight Area to Export"
+msgstr "Zone en surbrillance à exporter"
+
+#: src/Layouts/HeaderBar.vala:391
+msgid "Main Menu"
+msgstr "Menu principal"
+
+#: src/Layouts/HeaderBar.vala:468 src/Services/ActionManager.vala:224
+#: src/Services/ActionManager.vala:243 src/Services/ActionManager.vala:262
+#, c-format
+msgid "Unable to open file at '%s'"
+msgstr "Impossible d'ouvrir le fichier « %s »"
+
+#: src/Layouts/MainCanvas.vala:46
+msgid "Button was pressed!"
+msgstr "Un bouton a été appuyé !"
+
+#: src/Layouts/MainCanvas.vala:206
+msgid "Export completed!"
+msgstr "Exportation complète !"
```

Comment:
This should be
> Export terminé !

Reply:
I think "Exportation terminée !" is better. What do you think about it?